governance.Jenkinsfile

```diff
@@ -20,8 +23,17 @@ podTemplate(label: label) {
                             checkout scm
                         }
 
-                        stage("validate external links") {
-                            validateLinks('find . -name "*.md" | grep -v "vendor" | grep -v "./call-ec/web-ui/README.md"')
+                        // Enable on PR when internal links will be fixed
+                        if(isMaster || params.TRIGGER_FULL_VALIDATION) {
+                            stage("validate internal links") {
+                                validateLinks('--ignore-external', repositoryName)
+                            }
+                        }
+
+                        if(isMaster || params.TRIGGER_FULL_VALIDATION) {
```

Comment:
These two if clauses have the same condition. Did you mean not master in one of them?

Reply:
It is useful for TW to execute full validation on PR's.